clang/lib/CIR/CodeGen/CIRGenTypes.cpp

```diff
@@ -399,6 +399,15 @@ mlir::Type CIRGenTypes::convertType(QualType type) {
     break;
   }
 
+  case Type::ExtVector:
+  case Type::Vector: {
+    const VectorType *vec = cast<VectorType>(ty);
+    const mlir::Type elemTy = convertTypeForMem(vec->getElementType());
+    resultType = cir::VectorType::get(builder.getContext(), elemTy,
+                                      vec->getNumElements());
```

Comment:
Reflect changes to builders:


```suggestion
    resultType = cir::VectorType::get(elemTy, vec->getNumElements());
```